app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -446,11 +446,11 @@ public boolean onOptionsItemSelected(final MenuItem item) {
 
                 return true;
             }
-            case R.id.action_format_zimwiki:
-            case R.id.action_format_keyvalue:
-            case R.id.action_format_todotxt:
-            case R.id.action_format_plaintext:
-            case R.id.action_format_markdown: {
+            case R.string.action_format_zimwiki:
+            case R.string.action_format_keyvalue:
+            case R.string.action_format_todotxt:
+            case R.string.action_format_plaintext:
+            case R.string.action_format_markdown: {
```

Comment:
Switched all type formats to strings

Reply:
R.string is again a build-time generated, non constant value, not?